packedForest/src/forestTypes/binnedTree/zipClassAndValue.h

```diff
@@ -38,11 +38,11 @@ namespace fp{
 				}
 
 				inline bool checkInequality(const zipClassAndValue<int,double>& otherData){
-					return std::abs(featureVal - otherData.featureVal) > fpSingleton::getSingleton().returnDoubleEpsilon();
+					return std::abs(featureVal - otherData.featureVal) > std::numeric_limits<double>::epsilon() * 4 * std::abs(featureVal + otherData.featureVal);
```

Comment:
we might also want `fabs` here...?  it always returns a float...

Reply:
fabs == abs in c++.  Only different in C.